### PR TITLE
Update to android-maven-plugin 4.2.1

### PR DESCRIPTION
**Pull request by WonderCsabo**
Related to #1387.

### CODE REVIEW

File: AndroidAnnotations/functional-test-1-5/pom.xml [hunk 1]
```diff
@@ -155,9 +155,9 @@
 	<build>
 		<plugins>
 			<plugin>
-				<groupId>com.jayway.maven.plugins.android.generation2</groupId>
```
Comment (yDelouis) [inline AndroidAnnotations/functional-test-1-5/pom.xml]: This is the same plugin but they changed the groupId, right ?

Comment (WonderCsabo) [inline AndroidAnnotations/functional-test-1-5/pom.xml]: Indeed.

**Comment by WonderCsabo** [comment]
Rebased onto develop

**Comment by yDelouis** [comment]
Thanks.

**Pull request merged**